### ISSUE TITLE
Adding possibility to customise notifications sounds

### ISSUE THREAD

**Issue by NeoHuncho** [zidoro/pomatez#636]
I propose adding a feature for customizable notification sounds. I have a custom build with custom notification sounds and it makes such a difference.

If there's no opposition or ongoing similar work, I'll start developing this feature next week and submit a PR when read

**Comment by sekwah41** [comment]
No opposition, we need to change it from default windows sounds anyway ^.^

**Comment by sekwah41** [comment]
Ill assign it to you for now :)

**Comment by roldanjr** [comment]
I think it's related to these PRs #602 and #606

**Comment by sekwah41** [comment]
Yep these all relate though 602 is more related to changing the default sound for everyone
